shoop/themes/classic_gray/__init__.py

```diff
@@ -28,6 +28,18 @@ class ClassicGrayTheme(Theme):
         ("show_welcome_text", forms.BooleanField(required=False, initial=True, label=_("Show Frontpage Welcome Text"))),
     ]
 
+    def __init__(self, settings_obj=None):
+        super(ClassicGrayTheme, self ).__init__(settings_obj)
+        if "shoop.simple_cms" in settings.INSTALLED_APPS:
+            from shoop.simple_cms.models import Page
+            self.fields.append(
+                ("footer_cms_pages", forms.MultipleChoiceField(
+                    choices=Page.objects.filter(
```

Comment:
Wouldn't `Page.objects.translated(get_language())...` do the same thing?

Also, should this maybe use `ModelMultipleChoiceField`?